app/indexers/administrative_tag_indexer.rb

```diff
@@ -18,17 +18,20 @@ def initialize(id:, administrative_tags:, **)
   def to_solr
     Rails.logger.debug { "In #{self.class}" }
 
-    solr_doc = { 'tag_ssim' => [], 'exploded_tag_ssim' => [] }
+    solr_doc = { 'tag_ssim' => [], 'exploded_tag_ssim' => [], 'exploded_nonproject_tag_ssim' => [] }
     administrative_tags.each do |tag|
+      tag_prefix, rest = tag.split(TAG_PART_DELIMITER, 2)
+      prefix = tag_prefix.downcase.strip.gsub(/\s/, '_')
+
       solr_doc['tag_ssim'] << tag
       solr_doc['exploded_tag_ssim'] += exploded_tags_from(tag)
 
```

Comment:
no need for this space, IMO.

Reply:
fair enough, i think i was subconsciously separating the fields that might get removed soon from the new one that we'll be keeping

Reply:
we'll address this up when we remove exploded_tag_ssim.